src/libstore/globals.hh

```diff
@@ -956,6 +956,10 @@ public:
           resolves to a different location from that of the build machine. You
           can enable this setting if you are sure you're not going to do that.
         )"};
+
+    // The setting is named "registries" to keep compability with the old --no-registries flag
+    Setting<bool> useRegistries{this, true, "registries",
```

Comment:
This should be `use-registries` or something like that, since most of our boolean options are a verb (e.g. `keep-failed`). `registries` suggests that this is the list of registries.

Reply:
As noted in the comment, this is to keep compatibility with `--no-registries` option used in flake commands (such as `nix build --no-registries`). If you think it's better to keep both the option and a config setting, I can do that.

Reply:
Yeah, let's rename it to `use-registries` and provide `--no-registries` as a deprecated option that will be removed eventually.

Reply:
@edolstra done